app/experimenter/nimbus-ui/src/components/PageResults/GraphsWeekly/index.tsx

```diff
@@ -104,8 +105,8 @@ const GraphsWeekly = ({
 
   const [open, setOpen] = useState(false);
   const [embedded, setEmbedded] = useState(false);
-  const graphsVisibleClass = !open ? "d-none" : "";
-  const graphsHiddenClass = open ? "d-none" : "";
+  const graphsVisibleClass = classNames(!open && "d-none");
+  const graphsHiddenClass = classNames(open && "d-none");
```

Comment:
See [here](https://github.com/mozilla/experimenter/pull/6231#discussion_r690536055) for context (pun intended)